### PR TITLE
refactor/1517 - Bring back TNCM

### LINKED ISSUE #1517: Bring back TNCM

### PR DESCRIPTION
**Pull request by Fechulo**
Resolves #1517 

There's more invisible fixes than real ones 😩 

### [Test me here!](https://openscope-dev-pr-1519.herokuapp.com/)

### CODE REVIEW

**Comment by Fechulo** [comment]
I want to test it a little more before merging but it's ready for review

**Comment by Fechulo** [comment]
> I want to test it a little more before merging but it's ready for review

Ok, fully ready now 😄

File: assets/airports/tncm.json [hunk 1]
```diff
@@ -9,433 +10,3385 @@
...
+            "altitude": "",
+            "speed": "",
+            "method": "random",
+            "rate": 3.34,
+            "airlines": [
+                ["fdx/feeder", 3],
+                ["guy/at45", 3],
+                ["iwy", 3],
+                ["lia", 4],
+                ["n/lightGA", 1],
```
Comment (erikquinn) [inline assets/airports/tncm.json]: this fleet contains a reference to a BE36. There is a file for this, but the icao property of that file is BE20 (surely a copy-paste error). Could you verify the details are right for the BE36 (or adjust as necessary) and then rename the icao property to match?

Comment (Fechulo) [reply assets/airports/tncm.json]: I have no idea how long that's been like that 😬

**Comment by Fechulo** [comment]
Yeah, the 'hill' is disappointingly low actually. The highest point on the island, Pic Paradis, is just 1,391ft tall.

Anyway, ready for re-review!

**Pull request merged**